_includes/code/wcs.authentication.api.key.mdx

```diff
@@ -3,7 +3,34 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 <Tabs groupId="languages">
-<TabItem value="py" label="Python">
+<TabItem value="py4" label="Python (v4)">
+
+```python
+import weaviate
+
+# Connect to a WCS instance
+import os
+
+client = weaviate.connect_to_wcs(
+    cluster_url=os.getenv("WCS_DEMO_URL"),  # Set this environment variable
+    auth_credentials=weaviate.AuthApiKey(os.getenv("WCS_DEMO_RO_KEY"))  # Set this environment variable
+)
+
+# Connect to a local Weaviate instance
```

Comment:
I think this should be a separate snippet. 

Instead of showing all possible helper functions, about:

```python
import weaviate
import os

# Set your Weaviate API key
# Similar methods can be used for other helper functions where applicable (e.g. `.connect_to_custom`)
auth_credentials = weaviate.AuthApiKey("<YOUR_API_KEY>")  # Set this, or use an env var, e.g. os.getenv("WCS_API_KEY")

client = weaviate.connect_to_wcs(
    cluster_url="<YOUR_WCS_URL>",  # Set this, or use an env var, e.g. os.getenv("WCS_URL")
    auth_credentials=auth_credentials  # Provide your API Key to Weaviate
)